runtime/doc/options.txt

```diff
@@ -6275,7 +6276,7 @@ A jump table for the options with a short description can be found at |Q_op|.
 
 	Examples:
 	Emulate standard status line with 'ruler' set >vim
-	  set statusline=%<%f\ %h%m%r%=%-14.(%l,%c%V%)\ %P
+	  set statusline=%<%f\ %h%w%m%r%=%-14.(%l,%c%V%)\ %P
 <	Similar, but add ASCII value of char under the cursor (like "ga") >vim
 	  set statusline=%<%f%h%m%r%=%b\ 0x%B\ \ %l,%c%V\ %P
```

Comment:
The second example is also missing `%w` (only neovim has the second example).

Reply:
No, Vim also has it.

Reply:
Right. My bad